scripts/install_opengv.sh

```diff
@@ -1,11 +1,54 @@
 #!/bin/bash
 
-git clone https://github.com/laurentkneip/opengv.git "${opt_dir}/opengv"
+#--- Script for installing opengv ---#
 
+
+### Default imported variables setup ###
+
+if [ -z "${opt_dir}" ]; then
```

Comment:
Will these get annoying to write in every script? The default values are copied into N places. Imagine if we want to change one of these vars (like in the past, when we added `opt_dir`).

I think we should write `set -eux` and not have per-script defaults.

Reply:
That's a good point. I wanted to allow the scripts to run standalone without needing the values to be passed from `install_deps.sh` which isn't run from CI directly. Without these, I think `set -eux` would cause the Dockerfile script to fail. I think the best option for defaults now that these are in multiple files is to source them from a common bash script like the helper functions.